### DISCUSSION TITLE
milvus compatible gpu

### DISCUSSION THREAD

**Discussion by YangIsNotAvailable** [milvus-io/milvus#6235] [Q&A and General discussion]
Is milvus 1.1.1 compatible with Quadro cards or only Tesla?

**Comment by yhmo** [comment]
Milvus 1.x docker images is build on CUDA 10.1.
In this page: https://en.wikipedia.org/wiki/CUDA, it says "CUDA works with all Nvidia GPUs from the G8x series onwards, including GeForce, Quadro and the Tesla line", so I think Milvus 1.x can compatible with Quadro cards or only Tesla.
